include/repgp/repgp.h

```diff
@@ -59,155 +37,188 @@
 #include <stddef.h>
 #include <stdbool.h>
 #include <stdint.h>
-
 #include <json.h>
+
+#include <rnp/rnp_def.h>
 #include "repgp_def.h"
-#include "errors.h"
-
-struct rnp_key_store_t;
-typedef struct pgp_packet_t    pgp_packet_t;
-typedef struct pgp_cbdata_t    pgp_cbdata_t;
-typedef struct pgp_stream_t    pgp_stream_t;
-typedef struct pgp_reader_t    pgp_reader_t;
-typedef struct pgp_cryptinfo_t pgp_cryptinfo_t;
-typedef struct pgp_io_t        pgp_io_t;
-typedef struct pgp_key_t       pgp_key_t;
-typedef struct pgp_pubkey_t    pgp_pubkey_t;
-
-/** pgp_region_t */
-typedef struct pgp_region_t {
-    struct pgp_region_t *parent;
-    unsigned             length;
-    unsigned             readc; /* length read */
-    unsigned             last_read;
-    /* length of last read, only valid in deepest child */
-    unsigned indeterminate : 1;
-} pgp_region_t;
-
-void pgp_init_subregion(pgp_region_t *, pgp_region_t *);
-
-/** pgp_cb_ret_t */
-typedef enum { PGP_RELEASE_MEMORY, PGP_KEEP_MEMORY, PGP_FINISHED } pgp_cb_ret_t;
-typedef pgp_cb_ret_t pgp_cbfunc_t(const pgp_packet_t *, pgp_cbdata_t *);
+
+#define REPGP_HANDLE_NULL ((void *) 0)
```

Comment:
Better to use `NULL`, not 0, right?

Reply:
NULL definition is exactly same

Reply:
> NULL definition is exactly same.

Right but that's why we have `NULL`, right?. Is there an advantage to the method above? It just seems to obfuscate things and will break c++ code.

Reply:
How about we just change this to "NULL" and merge this?

Reply:
I think it is OK as is. NULL is implementation dependent.

Reply:
@flowher what's the advantage to using 0 here? I think we just want have it consistent. Very minor change, please? 😉